bin/net_clean

```diff
@@ -10,7 +10,7 @@ for image in forch/faucet forch/faux forch/gauge; do
 done
 
 echo Killing any old instances of forch...
-sudo kill `ps ax | fgrep forch | fgrep -v fgrep | awk '{print $1}'`
+sudo kill `ps ax | fgrep forch | fgrep -v fgrep | fgrep -v grep | fgrep -v net_clean | awk '{print $1}'` || true
```

Comment:
can you extract out the pid extraction part into a variable, or add line-breaks or something, this is getting ugly!

Reply:
Done.